pkg/apis/nmstate/v1/nodenetworkstate_types.go

```diff
@@ -357,3 +357,7 @@ const (
 type LinkAggregationMode string
 
 // TODO: add const strings
+
+func init() {
```

Comment:
this Whole file should go away and be created again, with proper review process. operator-sdk also adds some metadata that will be generated AFAIK. Would you mind dropping `pkg/apis` all together for the first PR?